openassessment/xblock/self_assessment_mixin.py

```diff
@@ -54,12 +54,14 @@ def self_path_and_context(self):
         context = {'allow_latex': self.allow_latex}
```

Comment:
Agreed - there's not a huge penalty when including it in the few cases it isn't needed. Let's just do it once for simplicity.